cmd/cli/plugin/package/README.md

```diff
@@ -146,7 +146,53 @@ Use "tanzu package repository [command] --help" for more information about a com
     namespace                   external-dns  string   The namespace in which to deploy ExternalDNS.
    ```
 
-1. Install a package
+   Example 3: Generate default values.yaml for a package
+
+   ```sh
+   >>> tanzu package available get contour.tanzu.vmware.com/1.18.2+vmware.1-tkg.1-rc.1 --generate-default-values-file
+    NAME:                             contour.tanzu.vmware.com
+    VERSION:                          1.18.2+vmware.1-tkg.1-rc.1
+    RELEASED-AT:                      2021-10-04 17:00:00 -0700 PDT
+    DISPLAY-NAME:                     contour
+    SHORT-DESCRIPTION:                An ingress controller
+    PACKAGE-PROVIDER:                 VMware
+    MINIMUM-CAPACITY-REQUIREMENTS:    Varies significantly based on number of Services, Ingresses/HTTPProxies, etc. A starting point is 128MB RAM and 0.5 CPU for each Contour and Envoy pod, but this can and should be tuned based on observed usage.
+    LONG-DESCRIPTION:                 An Envoy-based ingress controller that supports dynamic configuration updates and multi-team ingress delegation. See https://projectcontour.io for more information.
+    MAINTAINERS:                      [{Steve Kriss} {Steve Sloka} {Nick Young} {Sunjay Bhatia} {Nicholas Seemiller}]
+    RELEASE-NOTES:                    contour 1.18.2 https://github.com/projectcontour/contour/releases/tag/v1.18.2
+    LICENSE:                          [VMware’s End User License Agreement (Underlying OSS license: Apache License 2.0)]
+    SUPPORT:                          Support provided by VMware for deployment on TKG 1.4+ clusters. Best-effort support for deployment on any conformant Kubernetes cluster. Contact support by opening a support request via VMware Cloud Services or my.vmware.com.
+    CATEGORY:                         [ingress]
+
+    Created default values file at /home/contour-default-values.yaml
```

Comment:
The `/home` here probably means current directory?

Reply:
yes, it prints out absolute path.